code/rounds/PreRound.cs

```diff
@@ -69,5 +72,17 @@ public override void OnPlayerSpawn(TTTPlayer player)
 
             base.OnPlayerSpawn(player);
         }
+
+        private bool CheckMinimumPlayers()
+        {
+            if (Client.All.Count < TTTReborn.Gamemode.Game.TTTMinPlayers)
+            {
+                TTTReborn.Gamemode.Game.Instance.ChangeRound(new WaitingRound());
+
+                return false;
+            }
+
+            return true;
+        }
```

Comment:
We also have just duplicate methods redefined, is there a place where we could put these functions? maybe in `BaseRound`? Maybe as a static method?

Reply:
BaseRound would be my preferred way of checking.